### PR TITLE
Makes updates to README content

### PR DESCRIPTION
**Pull request by emileighoutlaw**
cc @ccostino

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,61 +87,74 @@ To disable, run:
...
+1. Read [Cloud Foundry documentation](https://docs.cloudfoundry.org/devguide/cf-cli/install-go-cli.html) to install Cloud Foundry CLI.
 
+2. Install autopilot by running:
 ```
 cf install-plugin autopilot -r CF-Community
 ```
+[Learn more about autopilot](https://github.com/concourse/autopilot).
+
+#### Deployment steps
+1. [Log in to Cloud Foundry](https://docs.cloudfoundry.org/cf-cli/getting-started.html). Org should be `fec`. Space can be `dev`, `stage`, or `prod`.
```
Comment (jmcarp) [inline README.md]: Running `invoke deploy` (step 3) automatically logs in to cloud.gov using the credentials set in step 2. I don't think this step is necessary.

File: README.md [hunk 2]
```diff
@@ -11,80 +11,70 @@
...
+    * Node.js (which includes npm)
+    * PostgreSQL (the latest 9.4 release). 
+         * Read a [Mac OSX tutorial](https://www.moncefbelyamani.com/how-to-install-postgresql-on-a-mac-with-homebrew-and-lunchy/) (or follow your OS package manager)
+         * Read a [Windows tutorial](http://www.postgresqltutorial.com/install-postgresql/)
+         * Read a [Linux tutorial](http://www.postgresql.org/docs/9.4/static/installation.html)
 
-Clone this repository.
 
-Create a Python environment for your project. We use virtualenv and virtualenv wrapper, but feel free to create your environment with your preferred set up.
+2. Set up your Node environment—  learn how to do this with our [Javascript Ecosystem Guide](https://pages.18f.gov/dev-environment-standardization/languages/python/).
```
Comment (LindsayYoung) [inline README.md]: This links to Python, and I think it should link to node

Comment (emileighoutlaw) [inline README.md]: Definitely right!

File: README.md (hunk 1 above)
Comment (emileighoutlaw) [inline README.md]: Ahh, that makes sense. @ccostino, this change sound good to you, too? If we all agree, I'll pop in and fix it!

File: README.md [hunk 3]
```diff
@@ -11,80 +11,70 @@
...
 
-<!--endrequirements-->
+### Project prerequisites
+1. Ensure you have the following requirements installed:
 
-## Quick(ish) start
+    * Python 3.4 (which includes pip and virtualenv)
+    * Node.js (which includes npm)
+    * PostgreSQL (the latest 9.4 release). 
+         * Read a [Mac OSX tutorial](https://www.moncefbelyamani.com/how-to-install-postgresql-on-a-mac-with-homebrew-and-lunchy/) (or follow your OS package manager)
```
Comment (ccostino) [inline README.md]: Whoops, that part in parentheses (`or follow your OS package manager`) should be next to Linux, not Mac. :-)

File: README.md [hunk 4]
```diff
@@ -11,80 +11,70 @@
...
-* npm
-* PostgreSQL
 
-<!--endrequirements-->
+### Project prerequisites
+1. Ensure you have the following requirements installed:
 
-## Quick(ish) start
+    * Python 3.4 (which includes pip and virtualenv)
+    * Node.js (which includes npm)
```
Comment (ccostino) [inline README.md]: Given that we list explicit versions for Python and PostGres, we should probably note that for Node.js too - I think including something like `; the latest LTS or stable release is fine` as the last part of what's inside the parentheses is fine.

File: README.md (hunk 1 above)
Comment (ccostino) [inline README.md]: Yes, thanks for the catch @jmcarp!

File: README.md (hunk 4 above)
Comment (emileighoutlaw) [inline README.md]: LTS?

Comment (ccostino) [inline README.md]: I'm sorry, I should've explained!  LTS means "Long Term Support", it's an acronym commonly used to refer to versions of software that have been ear-marked for extended maintenance and support cycles.  These versions are used when stability and security are mission-critical for an application or business.  In addition to Node.js having an LTS version available, the Django web framework and Ubuntu operating system also do.

**Comment by emileighoutlaw** [comment]
I believe I've captured all the wonderful copyedits y'all found, if someone wants to re-review and merge. Thanks!

**Comment by ccostino** [comment]
@emileighoutlaw:  Awesome, thank you!  There is only one thing left, which is removing step 1 from the deployment instructions and shifting the remaining steps up by one.

**Comment by ccostino** [comment]
Oh and we should reference https://github.com/18F/openFEC/issues/1559 with this PR.

**Comment by emileighoutlaw** [comment]
Okay! Really really ready this time ;)

This PR is the start of work on: #1559

File: README.md [hunk 5]
```diff
@@ -97,61 +87,70 @@ To disable, run:
...
+#### Deployment steps
+1. To deploy to Cloud Foundry, run 
 
 ```
-invoke deploy --space dev
+invoke deploy
 ```
 
-The `deploy` task will use the `FEC_CF_USERNAME` and `FEC_CF_PASSWORD` environment variables to log in.
-If these variables are not provided, you will be prompted for your Cloud Foundry credentials.
```
Comment (ccostino) [inline README.md]: Ack, sorry!  We lost these in the mix too but we should set them before the actual deployment steps.  Let's shift setting these two environment variables up to the `Deployment prerequisites` section as the last thing to do before the `Deployment steps`.  I'd probably do it like this:

    3. Set environment variables used by the deploy script:
    ```
    export FEC_CF_USERNAME=<your_cf_username>
    export FEC_CF_PASSWORD=<your_cf_password>
    ```

    If these variables are not set you will be prompted for your Cloud Foundry credentials when you deploy the app.

Comment (emileighoutlaw) [inline README.md]: Not to worry! I'm on it, and I'll ping you when done :)

**Comment by emileighoutlaw** [comment]
Wohoo! We have finished! @LindsayYoung, we'd love your eyes on this (no rush)

@noahmanger, I believe you were also interested in this (and you could tell us if it's human readable right now).

**Comment by ccostino** [comment]
@emileighoutlaw:  Looks like we have a merge conflict to resolve as well.  We can work through that tomorrow, it won't take long!

**Comment by emileighoutlaw** [comment]
👍

File: README.md [hunk 6]
```diff
@@ -7,360 +7,419 @@
...
-- [swagger](https://github.com/18F/swagger-ui):this generates our interactive API documentation
-- [openFEC-web-app](https://github.com/18f/openfec-web-app): the betaFEC web app for exploring campaign finance data
-- [fec-style](https://github.com/18F/fec-style): shared styles and user interface components
-- [fec-cms](https://github.com/18F/fec-cms): the content management system (CMS) for betaFEC
-- [proxy](https://github.com/18F/fec-proxy): this is a light weight app that coordinates the paths between the web app and CMS
+- [swagger-ui](https://github.com/18F/swagger-ui): forked repo that generates our interactive API documentation
+- [openFEC-web-app](https://github.com/18f/openfec-web-app): the web app for exploring campaign finance data
+- [fec-style](https://github.com/18F/fec-style): shared styles and user interface components, including this project's glossary and feedback tools
+- [fec-cms](https://github.com/18F/fec-cms): this project's content management system (CMS) 
+- [proxy](https://github.com/18F/fec-proxy): this is a lightweight app that coordinates the paths between the web app and CMS
```
Comment (adborden) [inline README.md]: Should the link text be `fec-proxy` to match the others in the list?

File: README.md [hunk 7]
```diff
@@ -7,360 +7,419 @@
...
 
 
 ## Get involved
-We’re thrilled you want to get involved!
+We welcome you to explore, make suggestions, and contribute to our code. 
 - Read our [contributing guidelines](https://github.com/18F/openfec/blob/master/CONTRIBUTING.md). Then, [file an issue](https://github.com/18F/fec/issues) or submit a pull request.
-- [Send us an email](mailto:betafeedback@fec.gov).
-- If you’re a developer, follow the installation instructions in the README.md page of each repository to run the apps on your computer.
+- If you'd rather send us an email, [we're thrilled to hear from you](mailto:betafeedback@fec.gov)!
```
Comment (adborden) [inline README.md]: Does 18F have a preference on the typography for apostrophe, i.e. `’` vs `'`

**Comment by adborden** [comment]
👍

File: README.md (hunk 7 above)
Comment (ccostino) [inline README.md]: Ack, I thought I removed all of the smart/smart-like quotes to make them plain text, good catch.

File: README.md (hunk 6 above)
Comment (ccostino) [inline README.md]: Yup!

**Comment by anthonygarvan** [comment]
:shipit:

**Comment by LindsayYoung** [comment]
We all seem to be in agreement, I will do the honors :)

**Pull request merged**